_posts/2019-04-24-gis-wisdom.md

```diff
@@ -0,0 +1,111 @@
+
+### 5. Editing
+{: .text-left}
+
+`refinement`
+
+The best datasets have lots of edits and editors.  They are continually being reviewed and refined. Think of multi-editing environments or Open Street Maps.  These datasets are so vast and valuable because they are 'live’ and diverse.  Everyone is bringing something unique to the product, which is difficult to produce with a small group of editors.
+
+But it's the refinement process that makes it valuable.  We too can continually refine through learning and sharing and practice and performance.  In turn, this advances our skills and adds to our value.
+
+Brush-up on some [essential editing vocabulary](http://desktop.arcgis.com/en/arcmap/10.3/manage-data/editing/essential-editing-vocabulary.htm).
+
+### 6. Domains
+{: .text-left}
+
+`consistency`
+
+Domains are set up in the database to establish known consistency. They provide quality assurance on the front-end while ensuring consistency on the back-end.  What makes domains so great is that the end-user can reliably expect certain values or results.  Developers and cartographers benefit from this as well. 
+
+Much like with domains, consistency can lead to reliable and expected results in human performance as well.  Think of athletes or musicians.  It's their consistency (and consequently, the establishment of muscle memory) that make the best so reliable and successful. Sure, repetition plays a role, but without pre-established methods (ie: domains) the results vary.   
+
+Learn more about [attribute domains](https://pro.arcgis.com/en/pro-app/help/data/geodatabases/overview/an-overview-of-attribute-domains.htm) and why they are so important. 
+
+### 7. Topology
+{: .text-left}
+
+`establishing principles`
+
+Topology is all about rules.  Essentially, it allows us to keep our data clean by defining a set of spatial expectations.  We are able to validate when needed and also allow for the inevitable exception.  Topology is so powerful because once you define the rules, you are then free to focus your efforts on other aspects of the data.
+
+Similarly, we also benefit from predefined rules -- or better yet, principles.  For example, think about driving an automobile.  We learn the rules-of-the-road before we get behind the wheel.  Why?  Because we don’t want to create the rules ‘on-the-fly’.  Knowing the rules-of-the-road in advance, allows us to focus on more enjoyable things, such as conversation or a good audiobook.
+
+Oddly, there's something freeing about having predetermined principles.  Knowing our principles allows us to then explore greater things.
+
+Learn the [basics of topology](https://pro.arcgis.com/en/pro-app/help/data/topologies/topology-basics.htm) and how it might assist you with data management.
+
+<br/>
+Drop me an [email](mailto:gbunce@utah.gov) If you can think of other forms of GIS wisdom.  I'll start a running list and post them below.
```

Comment:
I don't like having emails directly in posts. I would prefer we use [data contacts](https://github.com/agrc/gis.utah.gov#contact-information). Would you switch to that. Let me know if you need help.

Reply:
yup, I knew that.  oops.